torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -184,6 +183,10 @@ def filter_incompatible_and_dtype_convert_kwargs(kwargs):
                 continue
             else:
                 value = int(jit_type_utils.JitScalarType.from_dtype(value).onnx_type())
+        if key == "device":
+            # torch.device is not supported by onnxscript (no op). We turn it into
+            # a string.
```

Comment:
how will the `onnx_kwargs = {"device": str(value)}` be used later on when we call `        symbolic_fn = onnxfunction_dispatcher.dispatch(node=node, onnx_args=onnx_args, onnx_kwargs=onnx_kwargs, diagnostic_context=self.diagnostic_context,)`? Will it be just ignored or it will be part of the variant matching through attributes? That might be an useful information for the dostring of `filter_incompatible_and_dtype_convert_kwargs` and the PR description